lib/main.dart

```diff
@@ -10,7 +10,7 @@ Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp();
 
-  configureInjection();
+  await configureInjection();
```

Comment:
Again, no point in having a future if you are awaiting it in all scenarios, better to have it synchronous.

Reply:
I think this whole thing was necessary to avoid weird crashes, but I'll check again.